mypyc/transform/integer_optimization.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Dict
```

Comment:
Can you rename this to `mypyc.analysis.const_int`? `integer_optimization` doesn't describe what this does, but why, and we generally name things based on "what".